Configure

```diff
@@ -1193,24 +1195,23 @@ else               { $no_user_defines=1;    }
 
 # ALL MODIFICATIONS TO %config and %target MUST BE DONE FROM HERE ON
 
-unless ($disabled{engine}) {
-    $config{afalg}="";
-    if ($target =~ m/^linux/) {
-        my $minver = 4*10000 + 1*100 + 0;
-        if ($config{cross_compile_prefix} eq "") {
-            my $verstr = `uname -r`;
-            my ($ma, $mi1, $mi2) = split("\\.", $verstr);
-            ($mi2) = $mi2 =~ /(\d+)/;
-            my $ver = $ma*10000 + $mi1*100 + $mi2;
-            if ($ver < $minver) {
-                $disabled{afalg} = "too-old-kernel";
-            } else {
-                push @{$config{engdirs}}, "afalg";
-            }
+$config{afalg}="";
```

Comment:
That push line that I commented further down makes the connection.

Yes, having a `unless ($disabled{afalgeng})` wrapping around this would be useful.